### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
Some typos fixed for an already merged PR.

### CODE REVIEW

File: src/polycubed/src/polycubed.cpp [hunk 1]
```diff
@@ -53,6 +54,7 @@ std::shared_ptr<spdlog::logger> logger;
 // create core instance
 PolycubedCore *core;
 RestServer *restserver;
+int netlink_nofitication_id = -1;
```
Comment (frisso) [inline src/polycubed/src/polycubed.cpp]: netlink_nofitication_id  ==> netlink_notification_id
:-)

File: src/polycubed/src/service_controller.cpp [hunk 2]
```diff
@@ -284,5 +284,26 @@ bool ServiceController::parse_peer_name(const std::string &peer,
   return true;
 }
 
+void ServiceController::netlink_notification(int ifindex,
```
Comment (frisso) [inline src/polycubed/src/service_controller.cpp]: In general, can we add a little comment about what functions are about?

File: src/polycubed/src/polycubed.cpp (hunk 1 above)
Comment (mauriciovasquezbernal) [reply src/polycubed/src/polycubed.cpp]: Done,

File: src/polycubed/src/service_controller.cpp (hunk 2 above)
Comment (mauriciovasquezbernal) [reply src/polycubed/src/service_controller.cpp]: Sure,

**Comment by frisso** [comment]
@mauriciovasquezbernal Mauricio, unfortunately there are conflicts here; not sure if I originated the problem by merging the commits in the wrong order. Can you please have a look at it? Thanks!

**Comment by mauriciovasquezbernal** [comment]
@frisso https://github.com/polycube-network/polycube/pull/121 was not ready to be merged, that branch was based to an old version of this PR's branch that have some typos.
I'll change this PR just to fix the typos then.

**Pull request merged**